src/components/Modal/Modal.tsx

```diff
@@ -121,12 +120,11 @@ export function Modal(props: ModalProps) {
                 </span>
               </header>
               <main
-                ref={contentRef}
+                ref={modalBodyRef}
                 css={Css.fg1.overflowYAuto.if(hasScroll).bb.bGray200.if(!!forceScrolling).overflowYScroll.$}
               >
                 {/* We'll include content here, but we expect ModalBody and ModalFooter to use their respective portals. */}
                 {content}
-                <div ref={modalBodyRef} />
```

Comment:
Consolidated these refs. Didn't really seem necessary to need both. This also made it just simpler to have one less div of nesting for the ModalBody content.

Reply:
Nice